samples/angular/src/app/components/styleguide-sitecore-context/styleguide-sitecore-context.component.ts

```diff
@@ -23,7 +23,7 @@ export class StyleguideSitecoreContextComponent implements OnInit, OnDestroy {
 
   ngOnInit() {
     this.contextSubscription = this.jssContext.state.subscribe((state) => {
-      console.log('The current Sitecore Context in styleguide-sitecore-context.component.ts is...', state);
+      // console.log('The current Sitecore Context in styleguide-sitecore-context.component.ts is...', state);
```

Comment:
Do we want to remove this `console.log` completely?

Reply:
Maybe leave these in but change to `console.debug`?

Reply:
Re-introduced these as `console.debug`s